source/_guides/05-stack-configuration-best-practices.md

```diff
@@ -18,6 +18,23 @@ what they are and for what they're good and for what not. We're sure with this y
 At the end of the article, you'll find a complete example of all the recommendations here. You can pick the things you 
 want or apply to your use cases and leave the other things out. 
 
+### Get the "original" image without scaling
+
+While you can use the `dynamic` route without any parameters to get the image in the size as it was uploaded, we recommend
+to create a stack dedicated to that purpose. Create the stack without any operations. We recommend to enable the `autoformat`
```

Comment:
@ChristianRiesen do you have some more input that we can put here as to *why* we recommend to not use dynamic for this?

Reply:
I (who is not @ChristianRiesen ;) ) don't have any actual reason to not use the `dynamic` route. Especially for this (if you have complicated stacks, then `dynamic` can be long und ugly, but not in this case). Caching and such is all the same as with an actual stack.

Reply:
The argument is simply that having a stack means control over it. Dynamic is there regardless, so there is no control. Say you want to change an option or disallow that particualr part, you can remove, rename, and alter stacks. With dynamic it's rather not given (which defeats one of the original design principles behind rokka regarding caching and urls). Using stacks by default is a best practice, IMHO